doc/API/centreon-api-v2.yaml

```diff
@@ -1947,7 +2003,28 @@ components:
         hosts:
           type: object
           $ref: '#/components/schemas/Monitoring.Host_min'
-
+    Configuration.Proxy:
+      type: object
+      required:
+        - url
+        - port
+        - user
```

Comment:
is it possible to remove user/password by setting empty value ?

Reply:
I prefer to force the user to provide all parameters because I use the verb POST and not PATH.
The user will be able to define properties by empty value.